### PR TITLE
Add list screen

### PR DESCRIPTION
**Pull request by SultanArshad**

### CODE REVIEW

File: app-kotlin/src/main/res/layout/activity_main.xml [hunk 1]
```diff
@@ -1,9 +1,8 @@
...
+    android:id="@+id/home_container"
     xmlns:android="http://schemas.android.com/apk/res/android"
-    android:id="@+id/main_layout"
+    xmlns:tools="http://schemas.android.com/tools"
     android:layout_width="match_parent"
     android:layout_height="match_parent"
-    >
-
-</FrameLayout>
+    tools:context=".MainActivity"/>
```
Comment (xxfast) [inline app-kotlin/src/main/res/layout/activity_main.xml:8]: why do we need to bind this layout to the activity? 🤔

File: app-kotlin/src/main/res/layout/home_view_controller.xml [hunk 2]
```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent">
```
Comment (xxfast) [inline app-kotlin/src/main/res/layout/home_view_controller.xml:4]: do we need a constraint layout?

File: app-kotlin/src/main/res/values/dimens.xml [hunk 3]
```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="utf-8"?>
```
Comment (xxfast) [inline app-kotlin/src/main/res/values/dimens.xml:1]: for now, you can hardcode these dimensions in the layout themselves

File: app-kotlin/src/main/res/values/strings.xml [hunk 4]
```diff
@@ -1,3 +1,6 @@
 <resources>
   <string name="app_name">Training Kotlin</string>
+  <string name="attack">Attack</string>
+  <string name="image_of_the_pokemon">image of the pokemon</string>
+  <string name="pokemon_name">Pokemon Name</string>
```
Comment (xxfast) [inline app-kotlin/src/main/res/values/strings.xml:5]: Android apps tend to use title case for all their strings. 
```suggestion
  <string name="image_of_the_pokemon">Image of the pokemon</string>
  <string name="pokemon_name">Pokemon name</string>
```

**Review by xxfast — CHANGES_REQUESTED** [review]
Nice work 👍

File: app-kotlin/src/main/res/layout/activity_main.xml (hunk 1 above)
Comment (SultanArshad) [reply app-kotlin/src/main/res/layout/activity_main.xml:8]: thanks for pointing this out I had resolved this.

File: app-kotlin/src/main/res/layout/home_view_controller.xml (hunk 2 above)
Comment (SultanArshad) [reply app-kotlin/src/main/res/layout/home_view_controller.xml:4]: i had fixed this constraint layout by replacing all with frame layouts. in the latest pr #4

File: app-kotlin/src/main/res/values/strings.xml (hunk 4 above)
Comment (SultanArshad) [reply app-kotlin/src/main/res/values/strings.xml:5]: Thanks for the suggestion i had resolved this.

**Review by xxfast — APPROVED** [review]
Nice work 👍

**Pull request merged**